doc/classes/Vector2.xml

```diff
@@ -190,6 +190,15 @@
 				Returns the vector with all components rounded down (towards negative infinity).
 			</description>
 		</method>
+		<method name="from_angle" qualifiers="static">
+			<return type="Vector2">
+			</return>
+			<argument index="0" name="angle" type="float">
+			</argument>
+			<description>
+				Creates a [Vector2] from the given [code]angle[/code], so that [code]Vector2.from_angle(angle).angle()[/code] equals to [code]angle[/code].
```

Comment:
`Vector2.from_angle(angle).angle()` will return the `angle` wrapped into the `[-PI, PI]`  range so it might not be equal to the `angle`. Probably docs of `@GDscript.atan2`/`Vector2.angle` should mention their results are in `[-PI, PI]` range too.
It's also worth mentioning the `angle` is in radians.

Reply:
Well, they are equal assuming that PI equals to -PI (as this is equivalent vector). I should probably reword it 🙃

Reply:
Ok changed. I removed the `angle()` mention and just added a code sample.